inventory.yml.sample

```diff
@@ -74,20 +74,24 @@ all:
     # Local File Configuration #
     ############################
 
-    # Directory in which created/updated artefacts are placed
-    fetched_dest: ./fetched
+    root_path: /home/redhat/crucible/ # path to repository root
```

Comment:
I think that the name of the variable (`root_path`) has the potential to be misinterpreted, depending on the context and where it will be used.

Let's say someone decides to use it in the role for setting up the `http_store`. 
The name `root_path` is generic enough that it may mean all sorts of stuff. Is it the root path to the directory where registry files are stored? Or the root path to the systemd service definition files directory?

I'd be great if a user could choose any role at random, and without any prior knowledge of Crucible (including its inventory file), their ability to understand what the role does would not be hindered in any way. (You can image a user skimming through the code and coming across said variable; In my opinion, a proper reaction should be something like "oh, it's a Crucible-specific var, I may not know what it does, but at least it's not an important config var for the http_store, so I can keep reading")

We can make the variable more specific. Some suggestions that come to mind: `crucible_repo_root_path`, `playbooks_root_path`, etc. I favor the first candidate, but it's because I like prefixing in general, and so I'm deeply biased.

Reply:
Makes sense :) I think I prefer `repo_root_path`, as in principle something like DCI could import our roles (like they do with IPI) and then the crucible prefix could seem very odd.